deployment-local/local-tg

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+
+if [ "$1" = "startup-jenkins" ]; then
+    echo "__          _______  ____ ___    _______        _    _____      _     _    _       ____   _____   ___   _      "
+    echo "\ \        / / ____|/ __ \__ \  |__   __|      | |  / ____|    (_)   | |  | |    /  __  \/ _____|/    \| |     "
```

Comment:
The startup-jenkins was stuck in the middle without any errors/log output. We can improve this.

Reply:
I was supposed to set values for following two properties in params.properties. After setting that, server started fine. So, what's missing was the initial user experience.

```
uat_nexus_settings_path=uat-nexus.xml
common_configs_path=common-configs.properties
```

Reply:
I have added a log output.